geoslurp/discover/fesom/extractprofiles.py

```diff
@@ -1,20 +1,22 @@
 from netCDF4 import Dataset
 import numpy as np
 from netCDF4 import num2date
+from datetime import datetime
 
 def sec2month(nc_id):
     """convert seconds since.. to yyyy-mm-dd format"""
     times=nc_id["time"]
     units=times.units
     calendar=times.calendar
-    month=[entry.month for entry in num2date(times[:],units,calendar=calendar)]
-    year=[entry.year for entry in num2date(times[:],units,calendar=calendar)]
-    tt=[]
-    for j in range(len(month)):
-        if int(month[j])<10:
-            tt.append(str(int(year[j])) + '-0' + str(int(month[j])) + '-01')
-        else: 
-            tt.append(str(int(year[j])) + '-' + str(int(month[j])) + '-01')
+    tt=[entry for entry in num2date(times[:],units)]
+    # month=[entry.month for entry in num2date(times[:],units,calendar=calendar)]
+    # year=[entry.year for entry in num2date(times[:],units,calendar=calendar)]
+    # tt=[]
+    # for j in range(len(month)):
+    #     if int(month[j])<10:
+    #         tt.append(str(int(year[j])) + '-0' + str(int(month[j])) + '-01')
+    #     else: 
+    #         tt.append(str(int(year[j])) + '-' + str(int(month[j])) + '-01')
```

Comment:
Ok, I'm obviously doing this myself as well, but it's better to just toss out commented code. 
You can always get back to a certain commit if you want to reuse some code snippets